scispacy/custom_sentence_segmenter.py

```diff
@@ -31,32 +15,23 @@ def combined_rule_sentence_segmenter(doc: Doc) -> Doc:
 
     @param doc: the spaCy document to be annotated with sentence boundaries
     """
-    segmenter = pysbd.Segmenter(language='en', clean=False)
-    segments = segmenter.segment(doc.text)
-    segments = merge_segments(segments)
-
-    # pysbd splits raw text into sentences, so we have to do our best to align those
-    # segments with spacy tokens
-    segment_index = 0
-    current_segment = segments[segment_index]
-    built_up_sentence = ""
-    for i, token in enumerate(doc):
-        if i == 0 and (token.is_space or token.text == '.'):
-            token.is_sent_start = True
-            continue
-        if token.text.replace('\n', '').replace('\r', '') == '':
-            token.is_sent_start = False
-        elif len(built_up_sentence) >= len(current_segment):
+    segmenter = pysbd.Segmenter(language="en", clean=False, char_span=True)
+    sents_char_spans = segmenter.segment(doc.text)
```

Comment:
what is the new return type of `segment`?

Reply:
`TextSpan` object with (sent_str, start_char, end_char)